bip-musig2/reference.py

```diff
@@ -830,9 +830,9 @@ def test_sign_and_verify_random(iters: int) -> None:
 
         session_ctx = SessionContext(aggnonce, pubkeys, tweaks, is_xonly, msg)
         psig_1 = sign(secnonce_1, sk_1, session_ctx)
-        # Clear the secnonce after use to avoid accidental reuse
-        secnonce_1 = bytes(64)
         assert partial_sig_verify(psig_1, pubnonces, pubkeys, tweaks, is_xonly, msg, 0)
+        # An exception is thrown if secnonce_1 is accidentally reused
+        assert_raises(ValueError, lambda: sign(secnonce_1, sk_1, session_ctx), lambda e: True)
```

Comment:
That's good. Should we have signing test vectors for when k1 = 0 or k2 = 0?

Reply:
Good idea, done.